pom.xml

```diff
@@ -87,6 +91,16 @@
         <version>0.0.1-SNAPSHOT</version>
       </dependency>
       <dependency>
+        <groupId>io.servicecomb.saga.transports</groupId>
+        <artifactId>transport-httpclient-spring</artifactId>
```

Comment:
Does this module have something to do with spring?

Reply:
yes, to expose underlying transport implementation as a spring bean